aten/src/ATen/native/mps/operations/LinearAlgebra.mm

```diff
@@ -134,7 +134,7 @@ static void linalg_lu_factor_ex_out_mps_impl(const Tensor& A,
               "linalg.lu_factor(): MPS doesn't support complex types.");
   TORCH_CHECK(pivot, "linalg.lu_factor(): MPS doesn't allow pivot == False.");
 
-  Tensor A_t = A;
+  Tensor A_t = !A.is_contiguous() ? A.contiguous() : A;
```

Comment:
No need to do that, as the same check is part of `cotiguous()` op
https://github.com/pytorch/pytorch/blob/6ac8bc0cd2bfddb7985a858d49b6b5376fac212c/aten/src/ATen/native/TensorProperties.cpp#L115-L117
```suggestion
  Tensor A_t = A.contiguous();
```